index.html

```diff
@@ -19,20 +19,20 @@ <h1 id="title"> Microverse Survey Form</h1>
           <p id="description">Thank you for taking the time to help us improve the platform</p>
       </div>
 
-      <form action="" id="survey-form" class="form"> 
+      <form action="#" id="survey-form" class="form"> 
         <label for="name" id="name-label">Name</label>
-        <input type="text" name="" id="name" required placeholder="Enter your name">
+        <input type="text" id="name" required placeholder="Enter your name">
         
         <label for="email" id="email-label">Email</label>
         <input type="email" id="email" name="email" required placeholder="Enter your Email">
         
         <label for="number" id="number-label">Age (optional) </label>
         <input type="number" name="number" id="number" required placeholder="Age" min="10" max="100">
 
-        <label for="current-role">Which option best describes your current role?</label>
+        <label for="dropdown">Which option best describes your current role?</label>
 
         <select name="current-role" id="dropdown">
-          <option value=""></option>
+          <option value="">Choose your current position</option>
           <option value="Student">Student</option>
           <option value="Full Time Job">Full Time Job</option>
           <option value="Full Time Learner">Full Time Learner</option>
```

Comment:
Required changes:
- Your review was great and captured the main issues that needed some improvement. There was also a great explanation of why the changes got required. Keep it up.
-  [ ] I think we ought to have utilised the use of the [GitHub inline comments feature](/microverse/guides/tse/code_review/code_review_guidelines_2_0/-/blob/master/ADD_LINK_TO_VIDEO_OR_ARTCILE) at least once to leave review comments. Thereby, our code review could have come off as deliberate and complete.